README_zh-CN.md

```diff
@@ -146,6 +146,18 @@ MMSegmentation 是一个基于 PyTorch 的语义分割开源工具箱。它是 O
 
 请参考[快速入门文档](docs/zh_cn/get_started.md#installation)进行安装，参考[数据集准备](docs/zh_cn/dataset_prepare.md)处理数据。
 
+以下是安装的简要步骤。
+
+```shell
+conda create -n open-mmlab python=3.8 pytorch=1.10 cudatoolkit=11.3 torchvision -c pytorch -y
```

Comment:
`-c` might be too slow in mainland china, shall we add tips for tsinghua mirror in Chinese doc?